src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx

```diff
@@ -238,6 +238,9 @@ export const TitleHeader = styled(Text)<MainProps & ScalesWithCoverProps & Click
   ${({ main }) => main && fluidRange({ prop: 'fontSize', fromSize: '24px', toSize: '40px' })};
   line-height: ${({ main }) => (main ? 1 : 1.25)};
   cursor: ${(props) => (props.clickable ? 'pointer' : 'auto')};
+  max-width: 280px;
```

Comment:
This shouldn't be a constant value. Component will scale based on screen size so this should rather be expressed in percentage of the width

Reply:
To be honest, I struggled a bit to find which properties I need to change here 😄 . Should be good now